### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by AdamKasp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,18 @@
...
-
-We're using full-stack Behavior-Driven-Development, with [phpspec](http://phpspec.net) and [Behat](http://behat.org)
-
-Documentation
--------------
-
-Documentation is available at [docs.sylius.org](http://docs.sylius.org).
+All customization guides are available at [**This address**](https://docs.sylius.com/en/1.4/customization/index.html).
 
 Installation
```
Comment (CoderMaggie) [inline README.md]: Everything below is not needed

Comment (CoderMaggie) [reply README.md]: Okay, let's keep the License and Authors.

And before let's add links to Sylius/Sylius and Sylius/Sylius-Standard

File: README.md [hunk 2]
```diff
@@ -6,27 +6,18 @@
 
 <h1 align="center">Sylius Standard Edition</h1>
 
-<p align="center">This is Sylius Standard Edition repository for starting new projects.</p>
+<p align="center">This is Sylius Customization repository with all code uses in customization guides</p>
```
[the remark this answers is not in the document]
Comment (Zales0123) [reply README.md]: 😢

File: README.md [hunk 3]
```diff
@@ -4,71 +4,21 @@
...
-
-If you want to report a bug or suggest an idea, please use [GitHub issues](https://github.com/Sylius/Sylius/issues).
-
-Community Support
------------------
+Links
+-----
 
-Have a question? Join our [Slack](https://slackinvite.me/to/sylius-devs) or post it on [StackOverflow](http://stackoverflow.com) tagged with "sylius". You can also join our [group on Facebook](https://www.facebook.com/groups/sylius/)!
+[**sylius**](https://github.com/Sylius/sylius)
```
Comment (GSadee) [inline README.md]: sylius -> **S**ylius

File: README.md [hunk 4]
```diff
@@ -4,71 +4,21 @@
...
+<br>
+[**sylius-standard**](https://github.com/Sylius/sylius-standard)
```
Comment (GSadee) [inline README.md]: sylius-standard -> **S**ylius-**S**tandard

File: README.md [hunk 5]
```diff
@@ -4,71 +4,21 @@
...
-$ open http://localhost:8000/
-```
-
-Troubleshooting
----------------
+Customization Guide
+-------------------
 
-If something goes wrong, errors & exceptions are logged at the application level:
+All customization guides are available in the [**Sylius documentation: Customization Guide**](https://docs.sylius.com/en/1.4/customization/index.html).
```
Comment (GSadee) [inline README.md]: `latest` instead of `1.4`?

**Comment by GSadee** [comment]
Thanks Adam!

**Pull request merged**